runtime/test/planner-tests.js

```diff
@@ -194,7 +186,7 @@ describe('ConvertConstraintsToConnections', async() => {
     let { result, score } = results[0];
     assert.deepEqual(result.toString(),
 `recipe
-  map as view0
+  use as view0
```

Comment:
i think we don't have any tests with 'map' view left in this file. don't you should add some?